Source: Berd Steinhauser <berniyh@exherbo.org>
Reason: Make mandelbrot wallpaper plugin optional to avoid eigen dependency
Upstream: No

From 9505cb4bdf595181cf0af2cf475000b548a8b05a Mon Sep 17 00:00:00 2001
From: Bernd Steinhauser <berniyh@exherbo.org>
Date: Sat, 5 Sep 2009 15:37:56 +0200
Subject: [PATCH] Make mandelbrot optional.

---
 wallpapers/CMakeLists.txt |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/wallpapers/CMakeLists.txt b/wallpapers/CMakeLists.txt
index 21f1927..ca572e8 100644
--- a/wallpapers/CMakeLists.txt
+++ b/wallpapers/CMakeLists.txt
@@ -1,4 +1,4 @@
-add_subdirectory(mandelbrot)
+macro_optional_add_subdirectory(mandelbrot)
 add_subdirectory(pattern)
 add_subdirectory(weather)
 add_subdirectory(virus)
-- 
1.6.4.2

